etc/cin.profile

```diff
@@ -18,15 +18,15 @@ ipc-namespace
 net none
 nodbus
 nodvd
-nogroups
-nonewprivs
+#nogroups
+#nonewprivs
 notv
 noroot
 protocol unix
-seccomp
+#seccomp
 shell none
 
-private-bin cin,ffmpeg
+#private-bin cin,ffmpeg
```

Comment:
This is okay

Reply:
I'll run a small bench with seccomp enabled & disabled

Reply:
rendertime 

13:55  with     seccom
13:05  without it

Reply:
`seccomp` is indeed not for free performance-wise. The penalty is small, but when you do heavy computing, like rendering videos in your case, it could be measurable.

Reply:
yeah :)   

we need say if this 7% gap  its acceptable (i've a fx 8350),  MAY be small (2-3)  at an i5,  <1 at an i3 or memecpus,  huge (14) with a 2700X  and INSANELY BAD (24-36) with a TR/i9

Reply:
@bn0785ac Since `seccomp` is an important component of the sandbox, I'm not sure if disabling it is the right way. But probably there should be a comment in the profile, so other users can find the solution easily when they run into this issue.

I assume `nogroups` has a similar effect?

Reply:
i like this **accepted**  

with nogroups i found a harder gap  
the same file spent 16:05  (same setup, fx 8350-4.1) **idk if with an i3-i5 you'll notice a ligher gap, or if with a  r5-2600/600/ ryzen 7/TR/i9 you'll notice an insanely evil gap**

>15%

Reply:
**edit**: i've added this.

* re-enable seccomp

* add a comment: its an estimated due to  my benchs,  **as i dont have other pcs** i can't confirm that an i3 really has a 2% gap, a 2700X  has a 14-28% gap,  or a 1950X/7980XE really has **OFFENSIVE gaps** (+50%)